content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Building an Analytics Stack in 2020"
+author: "Nelson Auner"
+# I can't paint like y'all do!
+# cover: "/img/analysts_or_self_serve.png"
+tags: ["Analytics"]
+date: 2020-03-30T12:16:09-05:00
+draft: false
+---
+
+Congratulations, you are launching an analytics/data function! Maybe you are the first
+analytics hire at a brand new company, or perhaps you are bringing analytics to a mature
+company that historically lacked a comprehensive data strategy. You sold the vision and also put together a roadmap based on your experience and current best practices. What could go wrong (and who could argue that refactoring from a star to snowflake schema is an important company-wide goal)?
+
+The reality is that, like any other function, analytics and data exist within the
+context of an organization - an organization that has competing priorities,
+politics, and limited resourcing. I wrote this article to help you combine pragmatism with best practices and prepare for the discussions with engineering, finance, and possibly exec that you did not realize you needed to have. 
+
```

Comment:
I think it would be super helpful to briefly address the overarching effort to get buy in before going into specific tools. In particular, I'd call out

A) Understand who your senior leadership advocates are, and engage them early to get feedback on how best to get approval for these tools. Especially important if you're new to the company. Also, understand who the decider is so you know how much this stakeholder input actually changes your ability to accomplish what you need to (similar to your comment about just owning the BI tool decision, which is SPOT ON). 

B) Have an elevator pitch for 1-2 pressing company problems this stack will solve. Maybe it currently takes forever to get things done because the data team doesn't own the whole pipeline, and your pitch is that the right tools will accelerate the time from question to insight by giving the team the ability to see a project all the way through without bottlenecks. Maybe the company has chaotic reporting with inconsistent definitions, and you are pitching a stack that creates a true source of truth, and an engineering workflow to enable peer reviews of metric definitions. Find a story that resonates across the company, and it's easier to get folks on board with spending $X more for the right tool to accomplish that goal, rather than having folks worry that you just want the shiny tool because it makes your life marginally easier (though especially on very small data teams, that too can be a reasonable goal!).